projects/Mallard/src/hooks/use-image-paths.ts

```diff
@@ -20,9 +26,12 @@ const selectImagePath = async (
         source,
         path,
     )}`
-    const fs = FSPaths.media(localIssueId, source, path)
+
+    const fs = getFsPath(localIssueId, { source, path }, imageSize)
+    console.log(fs)
     const fsExists = await RNFetchBlob.fs.exists(fs)
     return fsExists ? fs : api
+    //should this be a file url
```

Comment:
Is this a question? Should this be answered or should we create another ticket for it?

Reply:
nope, that's jetsam